cmd/ocsp-updater/main.go

```diff
@@ -575,7 +575,7 @@ func setupClients(c cmd.OCSPUpdaterConfig, stats metrics.Statter) (
 
 	var pubc core.Publisher
 	if c.Publisher != nil {
-		conn, err := bgrpc.ClientSetup(c.Publisher)
+		conn, err := bgrpc.ClientSetup(c.Publisher, metrics.NewStatsdScope(stats, "OCSPUpdater"))
```

Comment:
We setup a scope with this same name below, in main. We should probably reuse that one. Also, looking at that I notice that we call ProfileCmd with "OCSP-Updater", which should probably be "OCSPUpdater".